django-backend/fecfiler/contacts/views.py

```diff
@@ -76,14 +79,23 @@ def individual_lookup(self, request):
         if q is None:
             return HttpResponseBadRequest()
 
+        tokens = list(filter(None, re.split('[^\\w+]', q)))
+        if (not tokens) or (len(tokens) < 1):
+            return HttpResponseBadRequest()
+        term = ('.*' + '.* .*'.join(tokens) + '.*').lower()
```

Comment:
I believe this brings us back into the world of the upper/lower problem.  I personally don't want to be in the business of designing search engines and I'd recommend that we stick to already developed solutions.

Reply:
I believe you mentioned before @dheitzer that adding complexity to the querystring parsing proportionally adds unexpected behavior

Reply:
Yes that's a good point @toddlees.  I thought I would need to change to this based on discussion with @mjtravers and seeing how fec.gov seems to handle delimiters and Aurelia's feedback but I'm open to however we want to handle this.  Maybe we can discuss/review Tues when Matt's back?

Reply:
<img width="451" alt="image" src="https://user-images.githubusercontent.com/97105825/190675322-11a06e0e-357f-4bc3-b52d-7976c19ca0e5.png">
<img width="241" alt="image" src="https://user-images.githubusercontent.com/97105825/190676279-9a2efb2f-0fc3-41f7-89d4-a5d1017f097a.png">

Splitting up the search by tokens seems to make names with apostrophe's unmatchable.

Reply:
It looks like fec splits up the tokens like you've got, so it's probably good that we match it in that sense.  If we can implement that without trading off matching names with `'`, that might be the way forward.

Reply:
In a meeting with @MitchellTCG  we decided that we will pass on matching names with `'` until later